test/unit/default-cache-middleware.js

```diff
@@ -49,14 +46,16 @@ describe('Default cache middleware', () => {
     const ctx = {
       req: {
         method: 'get',
-        uri: 'http://example.com/item'
+        url: 'http://example.com/item'
       },
       cacheStore
     };
-    const body = JSON.stringify(_.set({}, '_links.self.href', 'http://example.com/item'));
-    await cacheStore.set(ctx.req.uri, body);
+    const bodyObj = _.set({}, '_links.self.href', 'http://example.com/item');
+    const bodyStr = JSON.stringify(bodyObj);
+    await cacheStore.set(ctx.req.url, bodyStr);
     await middleware(ctx, () => next(ctx));
-    expect(await ctx.res.text()).to.equal(body);
+    expect(await ctx.res.json()).to.eql(bodyObj);
```

Comment:
eql()?   I couldn't find this matcher in the jest docs, what is it?

Reply:
tests under `test/unit/` are using chai, https://www.chaijs.com/api/bdd/#method_eql